arch/risc-v/src/common/riscv_addrenv.c

```diff
@@ -342,6 +380,13 @@ int up_addrenv_create(size_t textsize, size_t datasize, size_t heapsize,
       goto errout;
     }
 
+  if (copy_kernel_mappings(addrenv) < 0)
```

Comment:
ret = copy_kernel_mappings(addrenv);
if (ret < 0)
  {
      serr("ERROR: Failed to copy kernel mappings to new environment");
      goto errout;
  }

Reply:
No problem